app/templates/payment_type_list.twig

```diff
@@ -0,0 +1,33 @@
+{# paramters:
+	formId
+	appconfig
+	paymentTypesRequired (optional)
+	checkedPaymentType (optional)
+#}
+<ul class="horizontal clearfix payment-type-list">
+{% for paymentType, options in appconfig['payment-types'] %}
+	{# Not using if on loop to make sure loop.index stays the same #}
+	{% if options.enabled %}
+		<li
+		{% if loop.first %}
+			class="first"
+		{% elseif loop.last %}
+			class="last"
+		{% endif %}>
+			<input
+				id="{$ formId $}-payment-type-{$ loop.index $}"
+				type="radio"
+				class="payment-type-select"
+				name="zahlweise"
+				value="{$ paymentType $}"
+				title="{$ paymentType|trans({}, 'paymentTypes') $}"
+				{% if paymentTypesRequired %}required="required"{% endif %}
```

Comment:
You can drop this line and parameter in the templates. `required` is a vestige from the times when the form was validated with both HTML5 and JavaScript. Since HTML5 is such a mess (cross-browser incompatibilities, bad stylability) we dropped that validation method and its attributes.